engine/installation/linux/docker-ce/debian.md

```diff
@@ -31,7 +31,7 @@ Raspbian versions:
 - Jessie 8.0 (LTS) / Raspbian Jessie
 - Wheezy 7.7 (LTS)
 
-Docker CE is supported on both `amd64` (`x86_64`) and `armhf` architectures for Jessie and
+Docker CE is supported on both `x86_64`  (`amd64`)  and `armhf` architectures for Jessie and
```

Comment:
extra space, and I prefer "`x86_64` (including `amd64`)"

Reply:
Hi, @mstanleyjones , I'm happy to make the change but "including" implies that amd64 is a subset of x86_64 when it's really the same thing. Do you really want this change?

Reply:
It's not really technically 100% the same thing. AMD64 is compatible with the x86_64 instruction set, I believe. @andrewhsu can you shed some light?